Signal/src/ViewControllers/ConversationView/Cells/ConversationViewCell.m

```diff
@@ -35,53 +35,10 @@ - (void)setIsCellVisible:(BOOL)isCellVisible
     _isCellVisible = isCellVisible;
 
     if (isCellVisible) {
-        [self forceLayoutImmediately];
+        [self layoutIfNeeded];
```

Comment:
Reverting this change; I think I misdiagnosed the issue and it should be fixed by the other changes in this PR.

Reply:
🎉